app/models/manageiq/providers/ansible_tower/shared/provider.rb

```diff
@@ -58,8 +58,85 @@ def url=(new_url)
     default_endpoint.url = uri.to_s
   end
 
+  def create_default_data
+    with_provider_connection do |c|
+      ensure_credential(c)
+      organization = ensure_organization(c)
+      inventory = ensure_inventory(c, organization.id)
+      ensure_host(c, inventory.id)
+    end
+  end
+
+  def default_organization(connection = connect)
+    @@default_org ||= connection.api.organizations.all(
+      :name => product_name
```

Comment:
On second (third?) thought a flag won't really help, not for the organization at least, because we don't collect inventory for the organization.